### PR TITLE
HBASE-24113 Upgrade the maven we use from 3.5.4 to 3.6.3 in nightlies

### PR DESCRIPTION
**Pull request by saintstack**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/1/console in case of problems.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/1/console in case of problems.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/1/console in case of problems.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m  0s |  Docker mode activated.  |
| -1 :x: |  docker  |   0m  6s |  Docker failed to build yetus/hbase:97a47ed0ee.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hbase/pull/1430 |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/1/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m  0s |  Docker mode activated.  |
| -1 :x: |  docker  |   0m 32s |  Docker failed to build yetus/hbase:97a47ed0ee.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hbase/pull/1430 |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/1/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m  0s |  Docker mode activated.  |
| -1 :x: |  docker  |   0m 33s |  Docker failed to build yetus/hbase:97a47ed0ee.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hbase/pull/1430 |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/1/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: dev-support/docker/Dockerfile [hunk 1]
```diff
@@ -89,7 +89,7 @@ RUN curl --location --fail --silent --show-error --output /tmp/hadolint "${HADOL
   echo "${HADOLINT_SHA256} */tmp/hadolint" | sha256sum -c -
 
 FROM BASE_IMAGE AS MAVEN_DOWNLOAD_IMAGE
-ENV MAVEN_VERSION='3.5.4'
+ENV MAVEN_VERSION='3.6.3'
 ENV MAVEN_URL "https://archive.apache.org/dist/maven/maven-3/${MAVEN_VERSION}/binaries/apache-maven-${MAVEN_VERSION}-bin.tar.gz"
 ENV MAVEN_SHA256 'ce50b1c91364cb77efe3776f756a6d92b76d9038b0a0782f7d53acf1e997a14d'
```
Comment (ndimiduk) [inline dev-support/docker/Dockerfile:94]: Need to also update the artifact checksums.

Comment (saintstack) [reply dev-support/docker/Dockerfile:94]: Thanks

**Comment by saintstack** [comment]
Ad updating SHA's as per Nick.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/2/console in case of problems.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/2/console in case of problems.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/2/console in case of problems.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 51s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ branch-2 Compile Tests _ |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  hadolint  |   0m  3s |  There were no new hadolint issues.  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  hadoopcheck  |  10m 53s |  Patch does not cause any errors with Hadoop 2.10.0 or 3.1.2.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 19s |  The patch does not generate ASF License warnings.  |
|  |   |  19m 32s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1430 |
| Optional Tests | dupname asflicense hadolint shellcheck shelldocs hadoopcheck xml |
| uname | Linux be701076149b 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 1df979df18 |
| Max. process+thread count | 76 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/2/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) shellcheck=0.4.6 hadolint=1.17.5-0-g443423c |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by busbey** [comment]
Can we please update the expected maven version in the pom file and in [the ref guide suggestions|http://hbase.apache.org/book.html#build.basic] if we need these more recent maven versions?

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 47s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  6s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ branch-2 Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   7m  8s |  branch-2 passed  |
| +1 :green_heart: |  compile  |   3m 29s |  branch-2 passed  |
| +1 :green_heart: |  shadedjars  |   4m 52s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 57s |  branch-2 passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   5m 44s |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 18s |  the patch passed  |
| +1 :green_heart: |  javac  |   3m 18s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   4m 40s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 56s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  |  19m 12s |  root in the patch failed.  |
|  |   |  57m 21s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/2/artifact/yetus-jdk8-hadoop2-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1430 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 8b5d00fd614f 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 1df979df18 |
| Default Java | 1.8.0_232 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/2/artifact/yetus-jdk8-hadoop2-check/output/patch-unit-root.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/2/testReport/ |
| Max. process+thread count | 412 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/2/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 48s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  6s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ branch-2 Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   8m 28s |  branch-2 passed  |
| +1 :green_heart: |  compile  |   4m 29s |  branch-2 passed  |
| +1 :green_heart: |  shadedjars  |   5m 46s |  branch has no errors when building our shaded downstream artifacts.  |
| -0 :warning: |  javadoc  |   0m 21s |  root in branch-2 failed.  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   6m 46s |  the patch passed  |
| +1 :green_heart: |  compile  |   4m 11s |  the patch passed  |
| +1 :green_heart: |  javac  |   4m 11s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   5m 42s |  patch has no errors when building our shaded downstream artifacts.  |
| -0 :warning: |  javadoc  |   0m 20s |  root in the patch failed.  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 227m 24s |  root in the patch passed.  |
|  |   | 267m 15s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/2/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1430 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 215cf2a44c01 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 1df979df18 |
| Default Java | 2020-01-14 |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/2/artifact/yetus-jdk11-hadoop3-check/output/branch-javadoc-root.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/2/artifact/yetus-jdk11-hadoop3-check/output/patch-javadoc-root.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/2/testReport/ |
| Max. process+thread count | 4030 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/2/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by saintstack** [comment]
> Can we please update the expected maven version in the pom file and in the ref guide suggestions if we need these more recent maven versions?

We don't 'need' this version, not that I know of. Just having us run a more recent mvn on our infra.

Refguide has 'You must use at least Maven 3.0.4.' in it. I could file another issue to see if this still holds?

I don't see any enforcement of minimal mvn in pom. Do we have one? We should add the nice enforcer check https://stackoverflow.com/questions/32971660/enforce-a-minimum-version-of-maven ?

**Comment by saintstack** [comment]
The failure seems unrelated... Single test. Let me rerun to be sure.

Looks like it is pulling the new mvn:

```
19:41:04  Step 21/60 : ENV MAVEN_VERSION='3.6.3'
19:41:04   ---> Using cache
19:41:04   ---> 490d5e5a84b5
19:41:04  Step 22/60 : ENV MAVEN_URL "https://archive.apache.org/dist/maven/maven-3/${MAVEN_VERSION}/binaries/apache-maven-${MAVEN_VERSION}-bin.tar.gz"
19:41:04   ---> Using cache
19:41:04   ---> b779892c565b
19:41:04  Step 23/60 : ENV MAVEN_SHA512 'c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0'
19:41:04   ---> Running in faa62e4e492a
19:41:05   ---> Running in 47991cbdfe61
19:41:05  Removing intermediate container 47991cbdfe61
19:41:05   ---> 1b3d07c98f12
```

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/3/console in case of problems.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/3/console in case of problems.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/3/console in case of problems.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   3m 13s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ branch-2 Compile Tests _ |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  hadolint  |   0m  3s |  There were no new hadolint issues.  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  hadoopcheck  |  11m 55s |  Patch does not cause any errors with Hadoop 2.10.0 or 3.1.2.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 19s |  The patch does not generate ASF License warnings.  |
|  |   |  22m 17s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/3/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1430 |
| Optional Tests | dupname asflicense hadolint shellcheck shelldocs hadoopcheck xml |
| uname | Linux 7e9dca29bf72 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 1df979df18 |
| Max. process+thread count | 63 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/3/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) shellcheck=0.4.6 hadolint=1.17.5-0-g443423c |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   3m 58s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  5s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ branch-2 Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   7m 25s |  branch-2 passed  |
| +1 :green_heart: |  compile  |   4m 17s |  branch-2 passed  |
| +1 :green_heart: |  shadedjars  |   5m 54s |  branch has no errors when building our shaded downstream artifacts.  |
| -0 :warning: |  javadoc  |   0m 22s |  root in branch-2 failed.  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   7m  3s |  the patch passed  |
| +1 :green_heart: |  compile  |   4m 19s |  the patch passed  |
| +1 :green_heart: |  javac  |   4m 19s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   5m 51s |  patch has no errors when building our shaded downstream artifacts.  |
| -0 :warning: |  javadoc  |   0m 20s |  root in the patch failed.  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 273m 35s |  root in the patch passed.  |
|  |   | 315m 47s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/3/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1430 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 7358480d0711 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 1df979df18 |
| Default Java | 2020-01-14 |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/3/artifact/yetus-jdk11-hadoop3-check/output/branch-javadoc-root.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/3/artifact/yetus-jdk11-hadoop3-check/output/patch-javadoc-root.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/3/testReport/ |
| Max. process+thread count | 3555 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/3/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   3m 58s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  6s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ branch-2 Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   6m 19s |  branch-2 passed  |
| +1 :green_heart: |  compile  |   3m 22s |  branch-2 passed  |
| +1 :green_heart: |  shadedjars  |   4m 45s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   3m  6s |  branch-2 passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   5m 46s |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 22s |  the patch passed  |
| +1 :green_heart: |  javac  |   3m 22s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   4m 39s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   3m  3s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 274m 58s |  root in the patch passed.  |
|  |   | 316m 13s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.8 Server=19.03.8 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/3/artifact/yetus-jdk8-hadoop2-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1430 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux aa6dbfcb5764 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 1df979df18 |
| Default Java | 1.8.0_232 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/3/testReport/ |
| Max. process+thread count | 3454 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1430/3/console |
| versions | git=2.17.1 maven=(cecedd343002696d0abb50b32b541b8a6ba2883f) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**